src/Unleash/Internal/CachedFilesLoader.cs

```diff
@@ -72,7 +68,7 @@ public CachedFilesResult EnsureExistsAndLoad()
                 try
                 {
                     fileSystem.WriteAllText(toggleFile, string.Empty);
-                    result.InitialToggleCollection = null;
+                    result.InitialState = string.Empty;
```

Comment:
For other reviewers - looks super suspect but this is really just to get a test to shush. All internal so I don't think it's a biggie